linera-execution/src/execution.rs

```diff
@@ -495,4 +506,50 @@ where
         }
         Ok(applications)
     }
+
+    async fn process_subscriptions(
```

Comment:
It'd be good to have a comment on this method describing what it does - `process_subscriptions` doesn't say anything really.